data/debug-cli/docker_debug.yaml

```diff
@@ -0,0 +1,162 @@
+command: docker debug
```

Comment:
We should probably add a section somewhere about the requirement to be logged in with a subscription to Docker Desktop.